code/modules/roguetown/roguemachine/steward.dm

```diff
@@ -350,6 +351,17 @@
 	popup.set_content(contents)
 	popup.open()
 
+/obj/structure/roguemachine/steward/proc/job_filter(advj, j)
+	if(advj in excluded_jobs)
```

Comment:
this is a bit redundant because Wretches are Adventurers and they already get filtered out, but if a specific subclass of a job would preferably be filtered out later, or if Wretches become the only thing to get filtered later, it'll still work.